apps/block_scout_web/assets/js/omni.js

```diff
@@ -1,27 +1,52 @@
 // local cache to re-use data
 let txMapCache = {};
 
+const explorerMap = {
+  "scroll-sepolia": "https://sepolia-blockscout.scroll.io/tx/",
+  "arbitrum-goerli": "https://goerli.arbiscan.io/tx/",
+  "optimism-goerli": "https://goerli-optimism.etherscan.io/tx/",
+  "linea-goerli": "https://explorer.goerli.linea.build/tx/",
```

Comment:
The chain names should match the ones in the txs, so please confirm these are matching the ones we decided to use. @kevinhalliday @ttarsi

Reply:
confirming that `optimism-goerli` and `arbitrum-goerli` are valid, and we can add the other 2 when we're ready